README.md

```diff
@@ -143,8 +152,76 @@ Configuration format is described in detail in the
   indicates if the Tarantool repository should be enabled (for packages with
   open-source Tarantool dependency).
 
-**Note**: If an instance is mentioned in the `cartridge_replicasets` section,
-it should be configured in `cartridge_instances`.
+### Role tags
+
+This role tasks have special tags that allows to perform only secified actions.
+Tasks are running in this order:
+
+* `cartridge-instances` - install package, update instances config and restart instances;
+* `cartridge-replicasets` - configure replicasets;
+* `cartridge-config` - configure cluster, contains this tasks:
+  * configure authorization (if `cartridge_auth` is defined);
+  * patch application clusterwide config (if `cartridge_app_config` is defined);
+  * bootstrap Vshard (if `cartridge_bootstrap_vshard` is defined);
+  * manage cartridge failover (if `cartridge_failover` is defined).
+
+**Note**, that `cartridge-config` tasks would be skipped if no one of `cartridge_auth`, `cartridge_app_config`, `cartridge_bootstrap_vshard` and `cartridge_failover` variables is specified.
+
+### Example scenario
+
+Using `--tags` and `--limits` options you can manage cluster different ways:
```

Comment:
Create one example without `--tags`. 
First reading of this section was unobvious.